ext/storage_sources/s3_store/S3StoreLogSystem.cxx

```diff
@@ -0,0 +1,73 @@
+#include <aws/core/Aws.h>
+#include <aws/core/utils/logging/AWSLogging.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+
+#include "S3StoreLogSystem.h"
+
+#include <cstdarg>
+
+using namespace Aws::Utils;
+using namespace Aws::Utils::Logging;
+
+S3StoreLogSystem::S3StoreLogSystem(WT_EXTENSION_API *wtApi, uint32_t awsVerbose)
+{
+    static const std::map<int32_t, Aws::Utils::Logging::LogLevel> verbosityMapping = {
+      {-3, Aws::Utils::Logging::LogLevel::Error}, {-2, Aws::Utils::Logging::LogLevel::Warn},
+      {-1, Aws::Utils::Logging::LogLevel::Info}, {0, Aws::Utils::Logging::LogLevel::Info},
+      {1, Aws::Utils::Logging::LogLevel::Debug}};
+    if (verbosityMapping.find(awsVerbose) != verbosityMapping.end()) {
+        logLevel = verbosityMapping.at(awsVerbose);
+    } else {
+        logLevel = Aws::Utils::Logging::LogLevel::Error;
+    }
+    wt_api = wtApi;
```

Comment:
Use camel case for `wt_api`. You can use `this` to refer to a member of the class if names are the same.